charts/clearblade-iot-enterprise/charts/cb-console/templates/cb-console-deployment.yaml

```diff
@@ -20,6 +20,15 @@ spec:
       imagePullSecrets:
         - name: gcr-json-key
       {{- end }}
+      tolerations:
+      - key: node.kubernetes.io/not-ready
+        operator: Exists
+        effect: NoExecute
+        tolerationSeconds: 0
+      - key: node.kubernetes.io/unreachable
+        operator: Exists
+        effect: NoExecute
+        tolerationSeconds: 0
```

Comment:
Should we also put a `NoSchedule` effect for each of these tolerations?  Correct me if I'm wrong, but it seems like if we don't, it would still be possible for a pod to be scheduled onto the node, then immediately be evicted.

Reply:
in this gcloud doc, it says NoExecute will prevent pods from being scheduled on the tainted node, so I think we're good with just the NoExecute. https://cloud.google.com/kubernetes-engine/distributed-cloud/vmware/docs/how-to/node-taints